src/kOS/Module/kOSVesselModule.cs

```diff
@@ -19,6 +19,7 @@ public class kOSVesselModule : VesselModule
         private bool initialized = false;
         private Vessel parentVessel;
         private bool hasRemoteTech = false;
+        private int counterRemoteTechRefresh = 0;
```

Comment:
I'm going to make this change and try running with it like that during my stream, which should be starting in an hour.